bin/diagnostic.js

```diff
@@ -1 +1,12 @@
 'use strict';
+const ask = require('../lib/ask.js');
+
+let answer = 'x;
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (x > 1 && < 10) {
+   console.log("Guess again, and choose between 1 and 10");
+   break;
+  }
+}
```

Comment:
Rachel and I went through this one in our 1 on 1 today. I had just started trying to work through it during the diagnostic so definitely knew there were a lot of errors! 

The one thing I'm still a little unclear on though is at the end: 

answer = ask("Guess my secret? ")       <-- What is that doing?  Is it accounting for when the answer is correct? So kind of like "otherwise" if the above stuff is not true, that means they guessed the number.  To me, this wasn't even asked of us in the question within the diagnostic though, right? It seems like the code solution has a lot more in it than what was actually asked in the diagnostic question, or am I misunderstanding something? 

We can chat tomorrow about this too!